packages/desktop/App.svelte

```diff
@@ -90,10 +82,10 @@
 
         /* eslint-disable no-undef */
         // @ts-expect-error: This value is replaced by Webpack DefinePlugin
-        if (!devMode && get(appStage) === AppStage.PROD) {
-            await setAppVersionDetails()
-            pollCheckForAppUpdate()
-        }
+        // if (!devMode && get(appStage) === AppStage.PROD) {
+        //     await setAppVersionDetails()
```

Comment:
I think you should only uncomment the polling if it doesn't work

Reply:
The setAppVersionDetails is throwing an error too causing the whole function to cancel and not register electron events.